Blog/models.py

```diff
@@ -33,12 +33,13 @@ class Meta:
 class Post(ContentModelMixin):
     title = models.CharField(max_length=64, default="")
     tags = TaggableManager()
+    slug = models.SlugField(default='')
```

Comment:
A slug must be unique per model. You could make the field `unique`:  `slug = models.SlugField(default='', unique=True)`.